Src/DependencyCollector/Shared.Tests/Implementation/ProfilerHttpProcessingTest.cs

```diff
@@ -115,6 +115,27 @@ public void RddTestHttpProcessingProfilerOnExceptionForGetResponse()
         }
 
         /// <summary>
+        /// Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.
+        /// </summary>
+        [TestMethod]
+        [Description("Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.")]
+        [Owner("cithomas")]
+        [TestCategory("CVT")]
+        public void RddTestHttpProcessingProfilerOnWebExceptionForGetResponse()
+        {
+            var request = WebRequest.Create(this.testUrl);
+            this.httpProcessingProfiler.OnBeginForGetResponse(request);
+            Thread.Sleep(this.sleepTimeMsecBetweenBeginAndEnd);
```

Comment:
I would try removing the sleep, see if it was even needed, and if it was needed then look for a status change to watch for instead of sleeping.  Based on the method names I would not expect sleeping here to be needed.